assets/src/components/Lightbox/Lightbox.js

```diff
@@ -0,0 +1,114 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+import PhotoSwipe from '../../../../node_modules/photoswipe/dist/photoswipe.js';
+import PhotoSwipeUI_Default from '../../../../node_modules/photoswipe/dist/photoswipe-ui-default.js';
+
+// `items` should be an array of object with this shape:
+// [{ src, w, h, title }, ...]
```

Comment:
[a reflection rather than a remark] Maybe we can look into TypeScript at some point for cases like this to make it easier to define the expected shape of props.